docs/getting-started/install-zowe-sdks.md

```diff
@@ -86,14 +100,60 @@ npm install core-for-zowe-sdk.tgz
 
 ### Python SDK *technical preview*
 
-If you install Python SDK packages from the online registry, the required dependencies are installed automatically.
+If you install Python SDK packages from the [online registry](#installing-an-sdk-from-an-online-registry), the required dependencies are installed automatically. **[should we add this sentence to the Kotlin and Java sections above?]**
 
 If you download the Python SDK packages from Zowe.org, the downloaded folder contains dependencies that you must install manually. Extract the `WHL` files from the folder, copy the files to your project, and issue the following command for each dependency:
 
 ```
 pip install <fileName>.whl
 ```
 
-## Getting started
+## Installing a Zowe Client SDK
+
+To get started, import the SDK package to your project. You can pull the packages from an online registry, or download the packages from Zowe.org to install locally.
+
+### Installing an SDK from an online registry
+
+Pull the packages from an online registry such as npm or PyPi:
+
+1. In command-line window, navigate to your project directory. Issue the following command to install a package from the registry:
+
+   - To import a Node.js package: `npm install <PackageName>`
+   - To import a Python package: `pip install <PackageName>`
+
+      - `<packageName>`
+      The name of the SDK package that you want to install, such as `zos-files-for-zowe-sdk`.
+
+    The package is installed. Node packages are defined in `package.json` in your project. Python packages are installed by default to `$PYTHONPATH/Lib/site-packages` (Linux) or to the Python folder in your local `/AppData` folder (Windows). **[what about java and kotlin?]**
+
+2. **(Optional)** You might want to automatically update the SDK version when updates become available, or you might want to prevent automatic updates.
+
+    - To define the versioning scheme for Node packages, use [semantic versioning](https://docs.npmjs.com/about-semantic-versioning).
+
+    - To define versioning for Python packages, specify versions or version ranges in a `requirements.txt` file checked-in to your project. For more information, see [pip install](https://pip.pypa.io/en/stable/cli/pip_install/) in the pip documentation.
+
+    **[what about java and kotlin?]**
```

Comment:
@adam-wolfe - please respond

Reply:
I would keep the format we have here in line 133, and replace the `requirements.txt` with pom.xml and build.gradle using these links for more info:

In Maven, versioning ranges can be specified in the pom.xml file (https://cwiki.apache.org/confluence/display/MAVENOLD/Dependency+Mediation+and+Conflict+Resolution#DependencyMediationandConflictResolution-DependencyVersionRanges).

In Gradle, versioning ranges can be specified in the build.gradle file (https://docs.gradle.org/current/userguide/single_versions.html)